libsplinter/src/base62.rs

```diff
@@ -11,3 +11,77 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+#[cfg(feature = "circuit-template")]
+use std::str::FromStr as _;
+
+#[cfg(feature = "circuit-template")]
+/// Get the next base62 string of the same length; if the given string is the maximum value (all
+/// characters are `z`), return `None`.
+pub fn next_base62_string(string: &str) -> Option<String> {
+    if string.is_empty() {
```

Comment:
I think next_base62_string should check that the input string only contains base62 characters.

Reply:
Updated to check if string is base62 and return an error if it's not.